app/build.gradle

```diff
@@ -16,7 +16,6 @@ android {
     targetSdkVersion rootProject.ext.targetSdkVersion
     versionCode 1
     versionName "1.0"
-    multiDexEnabled true
     vectorDrawables.useSupportLibrary = true
```

Comment:
Can be dropped as well according to https://developer.android.com/guide/topics/graphics/vector-drawable-resources

Reply:
good catch. I removed it in the library already, removing from the demo app now.